src/organisation/models/update-pba-numbers.model.ts

```diff
@@ -0,0 +1,42 @@
+export class UpdatePbaNumbers {
+  public existingPbaNumbers: string[] = [];
+  public pendingAddPbaNumbers: string[] = [];
+  public pendingRemovePbaNumbers: string[] = [];
+
+  constructor(existingPbaNumbers: string[]) {
+    this.existingPbaNumbers = existingPbaNumbers;
+  }
+
+  get hasDisplayPbaNumbers(): boolean {
+    return this.displayPbaNumbers.length > 0;
+  }
+
+  get displayPbaNumbers(): string[] {
+    return this.existingPbaNumbers.filter(pba => this.pendingRemovePbaNumbers.indexOf(pba) === -1);
```

Comment:
Might want a null check in here seeing as `existingPbaNumbers` is `public` and doesn't have a protective setter method.